generators/app/index.js

```diff
@@ -56,8 +56,10 @@ module.exports = class extends Generator {
   }
 
   install() {
-    if (this.options['skip-install']) {
-      return;
+    if (!this.options['skip-install']) {
+      this.installDependencies({
```

Comment:
Jup. I thought it would be useful if the dependencies are added to the package.json in any case 😊